packages/aws-cdk-lib/aws-ecs/lib/ec2/ec2-task-definition.ts

```diff
@@ -26,13 +26,16 @@ export interface Ec2TaskDefinitionProps extends CommonTaskDefinitionProps {
    */
   readonly networkMode?: NetworkMode;
 
-  /**
-   * An array of placement constraint objects to use for the task. You can
+  /*** An array of placement constraint objects to use for the task. You can
    * specify a maximum of 10 constraints per task (this limit includes
    * constraints in the task definition and those specified at run time).
    *
+   * Note: The `distinctInstances()` method is not supported when creating
```

Comment:
I don't see how this sentence makes sense in the context of `placementContraints` docstring